resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesVolumeUtils.scala

```diff
@@ -45,13 +45,21 @@ object KubernetesVolumeUtils {
       val pathKey = s"$volumeType.$volumeName.$KUBERNETES_VOLUMES_MOUNT_PATH_KEY"
       val readOnlyKey = s"$volumeType.$volumeName.$KUBERNETES_VOLUMES_MOUNT_READONLY_KEY"
       val subPathKey = s"$volumeType.$volumeName.$KUBERNETES_VOLUMES_MOUNT_SUBPATH_KEY"
+      val labelsKey = s"$volumeType.$volumeName.label."
```

Comment:
- Change the variable name from `labelsKey` to `labelKey` to match the plural.
- Define and use `KUBERNETES_VOLUMES_LABEL_KEY` at `Config.scala` like

https://github.com/apache/spark/blob/98f0d9f32322074b01285f405c86df29997634a3/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala#L773